### ISSUE TITLE
Fullscreen button visibility should be determined by support of Fullscreen API, not max-width:768px

### ISSUE THREAD

**Issue by da70** [readium/readium-js-viewer#423]
Hi,

Right now `viewer.css` has this rule for disabling the fullscreen button on narrow screens:

```css
@media (max-width:768px){
  .navbar .btn-group > .btn.icon-full-screen
  {
    display:none;
  }
```

However the fullscreen functionality is actually dependent on whether or not the browser supports the Fullscreen API.  In `EpubReader.js`:

```javascript
    var toggleFullScreen = function(){

        if (!screenfull.enabled) return;

        screenfull.toggle();
    }
```

The screenfull.js module will not create the `screenfull` object on iOS devices because Fullscreen API is not yet supported there, so nothing will happen when tapping the fullscreen button on iPhone or iPad.

The iPad Retina and iPhone 6 Plus have screen widths greater than 768px, so the fullscreen button will appear on those devices, but won't work.

Also, if there is an Android device that has screen width of 768px or less, the fullscreen button would not appear, even though it would work and in fact would be very useful.

Unless there is some other rationale for using width as a check, could fullscreen button hiding behavior be changed so that the button is visible if it will work, but not be visible if it would not work?  Perhaps using `screenfull` object check or feature detection such as screenfull.js module itself uses.

Thanks!

**Comment by danielweck** [comment]
Agreed (especially the `screenfull` detection part). Thanks for your feedback!

**Comment by da70** [comment]
Sure thing.  One thing to note also is that changing from portrait to landscape can cause a device's viewport width to cross the 768px threshold. Our test iPad is 768px wide in portrait and 1024px in landscape, so the fullscreen button actually becomes visible in landscape. Just something to keep in mind in case users report contradictory experiences: e.g. two identical phones or tablets but one user sees the fullscreen button and the other doesn't.  Or, the same user claims that the fullscreen button sometimes appears and sometimes doesn't on the same device, not knowing that the orientation is significant.

**Comment by danielweck** [comment]
Well put. The `768px` width threshold may have been relevant at the time this conditional media query was originally introduced in Readium's CSS, but it certainly seems inappropriate now. Sure, the growing number of toolbar buttons has created additional clutter in the user interface, but there's probably a better way to address this problem.

**Comment by da70** [comment]
Ah okay, I see.  So this CSS rule was for reclaiming real estate on narrow screens, not necessarily for hiding the button on devices on which it wouldn't work.
I think reduce the clutter would be great.  Ironically, though, it's the devices that have small screens and limited real estate that could most benefit from fullscreen mode.  Fullscreen removes the browser chrome as well as any wrapper site that is embedding the reader (I guess this is because fullscreen does a frame break, which actually works out well for us as our site has a rather large banner that we should probably shrink down on mobile).  This leaves a lot more room for viewing text on a phone in portrait mode.

You're right it's probably going to be tricky to balance these two needs.
